src/interpreter/interpreter.h

```diff
@@ -18,17 +18,22 @@
 #define interpreter_interpreter_h
 
 #include "store.h"
+#include "support/result.h"
 
 namespace wasm {
 
 class Interpreter {
 public:
   // TODO: Methods to instantiate modules.
   // TODO: Methods to run exported functions.
+  Interpreter() { store = interpreter::WasmStore(); }
```

Comment:
`store` will be default-initialized by default, so we can just skip having an explicit constructor for now.
```suggestion
```

Reply:
done